app/views/console/settings/index.phtml

```diff
@@ -473,7 +473,33 @@ $smtpEnabled = $this->getParam('smtpEnabled', false);
                                     <input name="teamId" data-ls-attrs="id=leave-teamId-{{member.$id}}" type="hidden" data-ls-bind="{{console-project.teamId}}">
                                     <input name="membershipId" data-ls-attrs="id=leave-membershipId-{{member.$id}}" type="hidden" data-ls-bind="{{member.$id}}">
 
-                                    <button class="danger">Leave</button>
+                                    <button data-ls-if="1 === {{members.memberships.length}}" class="danger" disabled>Leave</button>
+                                    <button data-ls-if="1 < {{members.memberships.length}}" class="danger">Leave</button>
```

Comment:
This ls-if shouldnt be required. If there is member ID that is not yours, there are at least 2 members already

Reply:
We don't care about the member userId to block the `leave` button, we just want to make sure the project is not abandoned no matter who is the last user. You can leave the project if you want to. The user ID check is for redirecting the user to the console home after leaving the project instead of getting an errored project list since you don't have access to the project anymore.